3_javascript/javascript1/week3/excercisescript.js

```diff
@@ -0,0 +1,144 @@
+
+// __________________ The Last Class Excercises ______________________
+
+
+// 1.Let’s try to write a function that will check if an element is in an array…
+
+function checkIfArray(arr) {
+
+    const result = Array.isArray(arr);
+
+    if (result) {
+        console.log(`[${arr}] is an array.`);
+    } else {
+        console.log(`[${arr}] is not an array.`);
+    }
+
+}
+
+const fruitsArray = ["Apple", "Orange", "grape"];
+checkIfArray(fruitsArray);
+
+
+
+//2. Create an array of ingredients you would like to have on a pizza. Pick least favorite of them and remove it
+
+const pizzatopping = ["green pepper", "chedar cheez", "garlic", "oninon", "peperoni", "mashrooms", "ice berg"];
+console.log(pizzatopping)
+const lastItem = pizzatopping.pop();
+console.log(pizzatopping);
+
+
+
+
+//3. Create an array of objects representing 3 people, each with a name and an age. 
+// Find the average age of the people in the array.
+
+let peopleArrayOfObjects = [
+
+    {
+        name: "John",
+        age: 30,
+    },
+
+    {
+        name: "Robert",   // alert(peopleArrayOfObjects[2].name)  ===> Robert
+        age: 35,
+    },
+
+    {
+        name: "Sara",
+        age: 22,
+    }
+
+];
+console.log(peopleArrayOfObjects);
+
+
+// __________________ Class Excercise ______________________
+
+let pizza = {
+
+    name: "Peperoni",
+    description: "Pizza is in normal size and with extra Mayo sauce on",
+    price: "35 kr."
+}
+
+pizza.ingredients = ["peperoni", "pepper", "tomato", "cheez", "onion", "mashrooms"];
+
+let key = prompt("Do you want corn as topping on your pizza? \n Plz answer with Yes or No");
+// "error" "ReferenceError: topping is not defined   ????????????
+
+// access by variable
+alert(pizza[key]);   //?????
```

Comment:
not sure wht is the goal of this exercise, but this will throw an undefined if the user answers either "yes" or "no" 

because you no the key will have a value "true" lets say, so you are alerting

pizza["yes"] 

and there is no such key as "yes" in the object

```
let pizza = {

    name: "Peperoni",
    description: "Pizza is in normal size and with extra Mayo sauce on",
    price: "35 kr."
    ingredients = ["peperoni", "pepper", "tomato", "cheez", "onion", "mashrooms"];
}
```